### ISSUE TITLE
[Super resolution] image Resizing &low psnr value result

### ISSUE THREAD

**Issue by ryujaehun** [pytorch/examples#326]
https://github.com/pytorch/examples/blob/dcdabc22b305d2f2989c6f03570dfcd3919e8a5b/super_resolution/data.py#L41
I think resizing LANCZOS interpolation is better than default BILINEAR
`Resize(crop_size // upscale_factor,interpolation=Image.LANCZOS)`
__How does downsampling work in a normal SR?__

And In the Set5 dataset, I found that the psnr value is lower than the bicubic method.
Why..?

**Comment by chroneus** [comment]
yes, LANCZOS is better and PSNR result wich 
For data augmentation which aims to different scenarios, even better to use random choice of resize method.

**Comment by ryujaehun** [comment]
I did downsampling using the LANCZOS method. But, in [ESPCN](https://arxiv.org/abs/1609.05158),They used the Gaussian blur method. I can not be sure this is the reason, but the PSNR value is too low. In the Set5 dataset, I found that the psnr value is lower than the bicubic method.

**Comment by zpkosmos** [comment]
Excuse me, have you solve the problem of low psnr value result ? I tried to transform the 91 images training set into hdf5 file,and trained on it ,but still had 4 dB psnr loss than bicubic method, could you please give me some advises?

**Comment by ryujaehun** [comment]
Have you measured the PSNR of the entire RGB channel?

**Comment by zpkosmos** [comment]
Yes, but it still less than the bicubic method, I don't know what is the problem that make this method works worse than bicubic method,  I can't sure whether the problem is in training process, SR process or the model.

**Comment by ryujaehun** [comment]
If so, just use the y channel to see the results.

**Comment by Dae12-Han** [comment]
I also have the same result. I got the espcn psnr value is lower than the bicubic method. Is there any reason why this pytorch/examples espcn code has lower psnr???
